CHANGES.md

```diff
@@ -1,3 +1,57 @@
+# Next Release
```

Comment:
The assumes all changes will be in the "# Next Release" section and automatically replaces it with the release number and creates a new empty "# Next Release" section.